### DISCUSSION TITLE
rubymine update

### DISCUSSION THREAD

**Discussion by vjpatil2017** [Homebrew/discussions#833] [Everyday usage]
hi everyone,

I am new to homebrew. i installed rubymine IDE software using homebrew. Everytime I open rubymine it asks to update to new version. I tried to update it through homebrew upgrade command. The rubymine doesn't seem to get updated.
What am I doing wrong. or should i just update the IDE from the update option it provides.

Thank you.

**Comment by SMillerDev** [comment]
Because it provides an update option on its own homebrew won't automatically update. You can run `brew upgrade --greedy` to force it, or just use the provided updater.
